### PR TITLE
Updated german translations for color+cpu plugins.

### PR DESCRIPTION
**Pull request by ThomasVie**

### CODE REVIEW

File: plugin-cpuload/translations/cpuload_de.desktop [hunk 1]
```diff
@@ -1,13 +1,2 @@
...
-
-Name=Cpu monitor
-Comment=Displays the current CPU load.
-
-
-
-
-# Translations
-Comment[de]=Prozesslast-Monitor
 Name[de]=Prozessauslastung
```
Comment (palinek) [inline plugin-cpuload/translations/cpuload_de.desktop]: Is it an intention to have `Processauslastung` vs. `Processorauslastung` on different places?

Comment (ThomasVie) [inline plugin-cpuload/translations/cpuload_de.desktop]: You are absolutely right. Thanks!

**Comment by palinek** [comment]
@ThomasVie please, squash the commits

**Comment by ThomasVie** [comment]
> please, squash the commits
What would you like to be squashed?
One commit per pull request?
One file per commit?
In total more or less pull requests?
...

**Comment by palinek** [comment]
> What would you like to be squashed?

I meant the squashing 'Corrected Prozess -> Prozessor' into 'Updated german translations for the clock and cpuload plugins.' ...

**Comment by ThomasVie** [comment]
Will create a new branch.

**Pull request closed without merging**